src/main.rs

```diff
@@ -0,0 +1,31 @@
+extern crate clap;
+use clap::{App, Arg};
+extern crate calamine;
+
+fn main() {
+    let matches = App::new("Rate extractor")
+        .about("Program to extract the rate index")
+        .arg(
+            Arg::with_name("input")
+                .short("i")
+                .long("input-file")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("output")
+                .short("o")
+                .long("output-file")
+                .takes_value(true),
+        )
+        .get_matches();
+
+    let input_file = matches
+        .value_of("input")
+        .expect("provide a valid argument for input file");
+    let output_file = matches
+        .value_of("output")
+        .expect("provide a valid argument for output file");
```

Comment:
always use passive sentence while writing error messages. like here if you write Error while getting input_file that will be more professional